odd-platform-ui/src/components/DataEntityDetails/Metadata/MetadataValueEditor/MetadataValueEditor.tsx

```diff
@@ -112,12 +117,7 @@ const MetadataValueEditField: React.FC<MetadataValueEditFieldProps> = ({
           variant='main-m'
           label={labeled ? t('Value') : undefined}
           placeholder={labeled ? '' : defaultText}
-          type={
-            metadataType &&
-            [MetadataFieldType.INTEGER, MetadataFieldType.FLOAT].includes(metadataType)
-              ? 'number'
-              : 'text'
-          }
+          type={metadataType && inputTypes.includes(metadataType) ? 'number' : 'text'}
```

Comment:
type resolving.
`[MetadataFieldType.INTEGER, MetadataFieldType.FLOAT]` will be `("INTEGER" | "FLOAT")[]` type but we need  `MetadataFieldType[]`